packages/manager/modules/octavia-load-balancer/src/load-balancers/load-balancer/pools/detail/health-monitor/service.js

```diff
@@ -39,4 +40,33 @@ export default class OctaviaLoadBalancerHealthMonitorService {
       postDatas,
     );
   }
+
+  editHealthMonitorName(projectId, region, healthMonitor, name) {
+    const newHealthMonitor = {
+      ...healthMonitor,
+      name,
+    };
+    return this.editHealthMonitor(projectId, region, newHealthMonitor);
```

Comment:
You should be able to send only the name (the rest of the health monitor shouldn't be updated by the API)

Reply:
OK I will try :) there was a bug on API side and I was not able to do it but they work on a fix it should be good after